src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -151,6 +151,11 @@
 
 class Command(BaseCommand):
     help = "Seed database with initial roles."
+    parent_child_models = {
+        "project": "rulebook",
+        "activation": "rulebookprocess",
+        "activationinstance": "auditrule",
```

Comment:
AuditRule is registered as a normal model with organization as its parent. Doesn't jive with this line.